CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Change Log
```

Comment:
Is this the first time to make a change log in the docs repo?

Reply:
Yes.
A tag version was created, but there was no change log, so it is difficult to remember history and leave a log.
Added CHANGELOG.md file to leave logs in the future.